airbyte-integrations/connectors/source-postgres/src/test-integration/resources/expected_spec.json

```diff
@@ -246,8 +246,8 @@
               "plugin": {
                 "type": "string",
                 "title": "Plugin",
-                "description": "A logical decoding plugin installed on the PostgreSQL server. The `pgoutput` plugin is used by default. The `wal2json` plugin is deprecated and will soon be removed so it's not recommended to use. Read more about <a href=\"https://docs.airbyte.com/integrations/sources/postgres#step-2-select-a-replication-plugin\">selecting replication plugins</a>.",
-                "enum": ["pgoutput", "wal2json"],
+                "description": "A logical decoding plugin installed on the PostgreSQL server.",
+                "enum": ["pgoutput"],
                 "default": "pgoutput",
                 "order": 2
               },
```

Comment:
Same than before, adding options to the user to not being able to chose... not good 🤣 
```suggestion
```

Reply:
See https://github.com/airbytehq/airbyte/pull/23112#discussion_r1119018382